feature/KAS 4401 lijst styling

kobemertens opened the pull request:
- fix sanitization
- add styling for multi-level lists

kobemertens committed:
fix sanitization

kobemertens committed:
add styling for multi-level lists

ValenberghsSven committed:
Merge branch 'master' of github.com:kanselarij-vlaanderen/decision-report-generation-service into feature/KAS-4401-lijst-styling

ValenberghsSven reviewed (commented):
Check ticket comment.
Styling doesn't match current docs. Unsure if this is "ok" for the users.

kobemertens committed:
adds more style fixes

adds https://github.com/kanselarij-vlaanderen/frontend-kaleidos/pull/1947/files
and
https://github.com/kanselarij-vlaanderen/frontend-kaleidos/pull/1949/files

tdn committed:
removed min-width CSS to get better list outlines

tdn reviewed (approved):
I removed the `min-width` property on the list items, since this was causing unwanted spacing, and seemed to have no other effect...

As far as I can see, this is working well
